src/pages/Folder/components/FolderCategory/index.jsx

```diff
@@ -0,0 +1,49 @@
+import styles from "./styles.module.css";
+import useWindowSize from "hooks/useWindowSize";
+
+export default function FolderCategory({
+  folders,
+  selectedId,
+  onSelectedFolder,
+}) {
+  const [windowSize] = useWindowSize();
+
+  const handleClick = (e) => {
+    onSelectedFolder({ name: e.target.textContent, id: e.target.id });
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.tags}>
+        <span
+          className={`${styles.tag} ${
+            selectedId === null ? styles.selected : ""
+          }`}
+          id={null}
+          onClick={handleClick}
+        >
+          전체
+        </span>
+        {folders.map((folder) => (
+          <span
+            className={`${styles.tag} ${
+              folder.id === selectedId ? styles.selected : ""
```

Comment:
P2:
의도하신게 selectedId랑 folder.id가 일치할때 디자인이 변경되는 것이면 동등연산자로 하셔야 합니다.
selectedId는 문자이고, folder.id는 숫자입니다.

```suggestion
              folder.id === selectedId ? styles.selected : ""
```